web/src/containers/P2P/Offers/P2POffersModal.tsx

```diff
@@ -143,7 +143,7 @@ const P2POffersModal: FC<Props> = (props: Props): ReactElement => {
         } else {
             const available = wallets.find(w => w.currency === currencyCode.toLowerCase())?.balance;
             const low = side === 'buy' ? +lowLimit * +price : lowLimit;
-            const top = side === 'buy' ? +topLimit * +price : topLimit;
+            const top = side === 'buy' ? getTopLimit() * +price : getTopLimit();
```

Comment:
```suggestion
            const topLimit = getTopLimit()
            const top = side === 'buy' ? topLimit() * +price : topLimit();
```
to not to call func twice